identity.md

```diff
@@ -1,39 +1,38 @@
-In this tutorial, we'll run through the full story of a verifiable claim.  
-To do so, three actors will be involved: a <span class="label-role claimer">claimer</span>, an <span class="label-role attester">attester</span> and a <span class="label-role verifier">verifier</span>.   
-So we'll need to create three identities.  
+In this tutorial, we'll run through the full story of a claim.
 
-We'll come back to what each of these actors represent, but for now let's create their identities.   
+To do so, three actors will be involved: a <span class="label-role claimer">claimer</span>, an <span class="label-role attester">attester</span> and a <span class="label-role verifier">verifier</span>.   
+We'll come back to what each of these roles represent, but for now let's create the <span class="label-role claimer">claimer</span> identity and the <span class="label-role attester">attester</span> identity. You'll see that the <span class="label-role verifier">verifier</span> identity won't be needed.
 
-# 👤 Identity 
+# 👤 Identity
 
-In KILT, an identity is an object representing an entity - for example a person, an organization or even a machine.     
+In KILT, an identity is an object representing an entity - for example a person, an organization or even a machine.
```

Comment:
Suggestion: representing to representation

Reply:
Changed, I've made the sentence clearer with "that represents"; does that help? ("Representation" doesn't work since we want a verb here instead)

Reply:
Yes, it is good.